.scrutinizer.yml

```diff
@@ -1,11 +1,8 @@
 filter:
```

Comment:
do we need both scrutinizer AND styleci? it looks like scrutinizer is not active in thsi repository